src/main/frontend/components/query_table.cljs

```diff
@@ -42,9 +42,9 @@
 (defn- locale-compare
   "Use locale specific comparison for strings and general comparison for others."
   [x y]
-  (if (and (string? x) (string? y))
-    (.localeCompare x y (state/sub :preferred-language))
-    (< x y)))
+    (if (and (number? x) (number? y))
+      (< x y)
+      (.localeCompare (str x) (str y) (state/sub :preferred-language) (clj->js {:numeric true}))))
```

Comment:
Better to use `cond` as `localeCompare` is only for string

Reply:
I don't think another branch is needed since this is coercing with `str`. Merging but feel free to mention if we missed something